pkg/counter/cache_test.go

```diff
@@ -84,6 +84,92 @@ func TestRemoveMurFromCounterWhenIsAlreadyZeroAndNotInitialized(t *testing.T) {
 		HaveMasterUserRecordsPerDomain(toolchainv1alpha1.Metric{string(metrics.Internal): -1})
 }
 
+// spaces tests ----------
+
```

Comment:
the tests look good.
Could you please also update the tests related to the initialization/synchronization (at the end of this file) so they cover also this new counter?

Reply:
ok let me check!